rlog_generator/rlog_generator.py

```diff
@@ -159,10 +158,9 @@ def log_generator(pattern_conf):
                     es.index(index=elastic_config["elastic_index"], pipeline=elastic_config["elastic_pipeline"], body={"message": log_str})
                 except Exception as e:
                     log.error(f"Error sending data to Elasticsearch: {e}")
-            
-            if log_output:                        
+
+            if log_output:
                 with open(path, "a") as f:
-                    template = random.choice(pattern_conf["template"])
```

Comment:
this was already called at line 151 `template = random.choice(pattern_conf["template"])` within the outer loop, but it got called again for the log output?